From 2df11aafe4eae33aab06d902ddeed6bd996b1575 Mon Sep 17 00:00:00 2001
From: Aaron Plattner <aplattner@nvidia.com>
Date: Mon, 16 Jun 2008 10:07:07 -0700
Subject: [PATCH] GeForce GTX 280 and 260 chip names.

Note: These GPUs already worked with this driver, this change just adds their
names to the table.
---
 src/nv_driver.c |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/src/nv_driver.c b/src/nv_driver.c
index a6202ce..d710d06 100644
--- xf86-video-nv-2.1.9/src/nv_driver.c
+++ xf86-video-nv-2.1.9/src/nv_driver.c
@@ -113,7 +113,7 @@ _X_EXPORT DriverRec NV = {
 #endif
 };
 
-/* Known cards as of 2008/06/05 */
+/* Known cards as of 2008/06/16 */
 
 static SymTabRec NVKnownChipsets[] =
 {
@@ -389,6 +389,8 @@ static SymTabRec NVKnownChipsets[] =
   { 0x10DE042D, "Quadro FX 360M" },
   { 0x10DE042E, "GeForce 9300M G" },
   { 0x10DE042F, "Quadro NVS 290" },
+  { 0x10DE05E1, "GeForce GTX 280" },
+  { 0x10DE05E2, "GeForce GTX 260" },
   { 0x10DE0600, "GeForce 8800 GTS 512" },
   { 0x10DE0602, "GeForce 8800 GT" },
   { 0x10DE0604, "GeForce 9800 GX2" },
-- 
1.5.3.7

